pipeline/schema.sql

```diff
@@ -1,4 +1,7 @@
---Script to delete foreign key constraints to drop tables.
+--Set default schema to allocated schema
+ALTER USER [alpha] WITH DEFAULT_SCHEMA = alpha;
```

Comment:
Just for the notes that this allows for accidentally skipping off the schema prefix